samples/highcharts/accessibility/accessible-pattern-fills/demo.js

```diff
@@ -0,0 +1,396 @@
+    },
+
+    xAxis: {
+        visible: false
+    },
+
+    yAxis: {
+        visible: false
+    },
+
+    exporting: {
+        enabled: false
+    },
+
+    series: [{
+        showInLegend: true,
+        label: {
+            enabled: false
+        },
+        type: 'column',
+        borderColor: Highcharts.getOptions().colors[0],
+        data: [{
+            y: 1,
+            color: {
+                patternIndex: 0
+            }
+        }, {
+            y: 1,
+            color: {
+                patternIndex: 1
+            }
+        }, {
+            y: 1,
+            color: {
+                patternIndex: 2
+            }
+        }, {
+            y: 1,
+            color: {
+                patternIndex: 3
+            }
+        }, {
+            y: 1,
+            color: {
+                patternIndex: 4
+            }
+        }, {
+            y: 1,
+            color: {
+                patternIndex: 5
+            }
+        }, {
+            y: 1,
+            color: {
+                patternIndex: 6
+            }
+        }, {
+            y: 1,
+            color: {
+                patternIndex: 7
+            }
+        }, {
+            y: 1,
+            color: {
+                patternIndex: 8
+            }
+        }, {
+            y: 1,
+            color: {
+                patternIndex: 9
+            }
+        }],
+        dataLabels: {
+            connectorColor: Highcharts.getOptions().colors[0],
+            formatter: function () {
+                var i = this.point.index;
+                return i > 9 ?
+                    'Custom pattern' : // For the last one, show custom label
+                    'default-pattern-' + i; // Show default pattern label
+            }
+        }
+    }]
+});
+
+function getColorPattern(i) {
+    var colors = Highcharts.getOptions().colors,
+        patternColors = [colors[2], colors[0], colors[3], colors[1], colors[4]],
+        patterns = [
+            'M 0 0 L 5 5 M 4.5 -0.5 L 5.5 0.5 M -0.5 4.5 L 0.5 5.5',
+            'M 0 5 L 5 0 M -0.5 0.5 L 0.5 -0.5 M 4.5 5.5 L 5.5 4.5',
+            'M 1.5 0 L 1.5 5 M 4 0 L 4 5',
+            'M 0 1.5 L 5 1.5 M 0 4 L 5 4',
+            'M 0 1.5 L 2.5 1.5 L 2.5 0 M 2.5 5 L 2.5 3.5 L 5 3.5'
+        ];
+
+    return {
+        pattern: {
+            path: patterns[i],
+            color: patternColors[i],
+            width: 5,
+            height: 5
+        }
+    };
+}
+
+Highcharts.chart('container3', {
+    chart: {
+        type: 'pie',
+        style: {
+            fontFamily: 'Roboto,Arial'
+        }
+    },
+
+    title: {
+        text: ''
+    },
+
+    exporting: {
+        enabled: false
+    },
+
+    tooltip: {
+        valueSuffix: '%',
+        borderColor: '#8ae'
+    },
+
+    plotOptions: {
+        series: {
+            label: {
+                enabled: false
+            },
+            dataLabels: {
+                enabled: false,
+                connectorColor: '#777',
```

Comment:
When dataLabels are disabled we don't need these other options (`connectorColor` and `format`)